_potd_171_enemies/sasquatch.md

```diff
@@ -68,11 +68,7 @@ job_specifics:
       - "Steel recommended any time, but required in Gloom, HP Penalty, and/or
       Item Penalty enchantments"
       - "Strength recommended"
-      - note: "Doable without steel IF"
-        subnotes:
-          - "There is no Gloom, HP Penalty, or Item Penalty enchantment"
-          - "Dreadwyrm or Titan and Garuda are up"
-          - "You have room to kite and sprint/shield before the pull"
+      - "Doable without steel IF. There is no Gloom, HP Penalty, or Item Penalty enchantment. Dreadwyrm or Titan and Garuda are up. You have room to kite and sprint/shield before the pull"
```

Comment:
The previous formatting was preferable

Reply:
I have no idea why it's been changed as I haven't touched it. Probably a previous pull from the original ODD fork, my msitake.